lib/Zonemaster/CLI.pm

```diff
@@ -278,11 +278,32 @@ has 'sourceaddr' => (
     isa           => 'Str',
     required      => 0,
     documentation => __(
-            'Source IP address used to send queries. '
+            'Deprecated (planned removal: v2024.1). '
+          . 'Source IP address used to send queries. '
           . 'Setting an IP address not correctly configured on a local network interface causes cryptic error messages.'
     ),
 );
 
+has 'sourceaddr4' => (
+    is            => 'ro',
+    isa           => 'Str',
+    required      => 0,
+    documentation => __(
+            'Source IPv4 address used to send queries. '
+          . 'Setting an IPv4 address not correctly configured on a local network interface fails silently.'
```

Comment:
Add 'Setting an IP address not correctly configured on a local network interface causes cryptic error messages.' and "Cannot be combined with --sourceaddr".

Reply:
> Add 'Setting an IP address not correctly configured on a local network interface causes cryptic error messages.'

From #316 I understand that there won't be such error messages. Do I understand it correctly?

Reply:
True, but I still think we should have the warning. Suggestion:
```
Setting an IP address not correctly configured on a local network interface will silently
prevent Zonemaster-CLI from working correctly.
```

Reply:
Yes. And your text is fine.